drivers/kw2xrf/Kconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2022 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_KW2XRF
+    bool "KW2XRF radio"
```

Comment:
Maybe you should be using the:
```kconfig
    bool
    prompt "KW2XRF radio" if !(MODULE_NETDEV_DEFAULT && HAVE_KW2XRF)
    default (MODULE_NETDEV_DEFAULT && HAVE_KW2XRF)
```

as this is a module with defaults and options?

Reply:
Sure, fixed and removed the `select` from the feature